package.json

```diff
@@ -17,8 +24,17 @@
     "fs-extra": "^9.1.0"
   },
   "devDependencies": {
-    "eslint": "^8.5.0",
+    "@types/fs-extra": "^11.0.1",
+    "@types/node": "^18.11.18",
+    "@typescript-eslint/eslint-plugin": "^5.49.0",
+    "@typescript-eslint/parser": "^5.49.0",
+    "eslint": "^8.32.0",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-plugin-import": "^2.25.3"
+    "eslint-plugin-import": "^2.27.5",
+    "parcel": "^2.8.3",
+    "typescript": "^4.9.4"
```

Comment:
Use fixed versions to ensure that project will be working next time one of those libraries will be updated